### DISCUSSION TITLE
红队行动守则 | 倾旋的博客

### DISCUSSION THREAD

**Discussion by giscus** [Rvn0xsy/rvn0xsy.github.io#23] [General]
# 红队行动守则 | 倾旋的博客

这篇文章是收集了多数红队在进行红队行动的经验之谈，由于工作发展原因，不再继续红队相关的工作，因此决定分享出来。

https://payloads.online/article/0ba7791c-bb0f-42e7-8623-22fe07b59fa8

**Comment by yangzhao917** [comment]
质量很高

**Comment by 0ldbaby** [comment]
👍
